snappy_pipeline/apps/tpls/project_config.yaml

```diff
@@ -4,18 +4,46 @@
 
 # Step Configuration ==============================================================================
 #
-# Configuration for paths with static data.  This has been preconfigured for the paths on the BIH
-# cluster.
+# Configuration for paths with static data (GRCh37/hs37d5 genome release, no CHR prefix in contig names).
+# This has been preconfigured for the paths on the BIH cluster.
 #
 static_data_config:
   cosmic:
-    path: /fast/projects/cubit/current/static_data/db/COSMIC/v72/GRCh37/CosmicAll.vcf.gz
+    path: /data/cephfs-1/work/projects/cubit/current/static_data/db/COSMIC/v72/GRCh37/CosmicAll.vcf.gz
   dbnsfp:
-    path: /fast/projects/cubit/current/static_data/db/dbNSFP/2.9/dbNSFP2.9.txt.gz
+    path: /data/cephfs-1/work/projects/cubit/current/static_data/db/dbNSFP/2.9/dbNSFP2.9.txt.gz
   dbsnp:
-    path: /fast/projects/cubit/current/static_data/db/dbSNP/b147/GRCh37/All_20160408.vcf.gz
+    path: /data/cephfs-1/work/projects/cubit/current/static_data/db/dbSNP/b147/GRCh37/All_20160408.vcf.gz
   reference:
-    path: /fast/projects/cubit/current/static_data/reference/GRCh37/hs37d5/hs37d5.fa
+    path: /data/cephfs-1/work/projects/cubit/current/static_data/reference/GRCh37/hs37d5/hs37d5.fa
+  features:
+    path: /data/cephfs-1/work/projects/cubit/current/static_data/annotation/GENCODE/19/GRCh37/gencode.v19.annotation.gtf
+
+# Step Configuration ==============================================================================
```

Comment:
While we're at it, we can correct the headers, which all just say "Step Configuration"